partner_kind/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,16 @@
+This module add kind of partner to partners.
+In the method _get_partner_kind : there si the selection you want to add as partner kind. To mange this générique module, we add 4 selections which can obviously be removed.
+In this selection field you will find :
+- The recordset address : for the company or person linked to a parent
+- the recordset user : for company internal person which cannot be concerned by the kind
+- the recordset company : for the other case if it's a company
+- the recordset person : as the field is required, there is a default value
```

Comment:
They are not recordset, just selection options, please change it in all the readme. 
Maybe " - Address : for the company or person linked to a parent" is enough.
Also, you can use the interface term for users.

Reply:
ok